source/target/freescale/kw24d/target.c

```diff
@@ -26,11 +26,13 @@
 
 // target information
 target_cfg_t target_device = {
-    .sector_size    = 2048,
-    .sector_cnt     = (KB(512) / 2048),
-    .flash_start    = 0,
-    .flash_end      = KB(512),
-    .ram_start      = 0x1FFF8000,
-    .ram_end        = 0x20008000,
-    .flash_algo     = (program_target_t *) &flash,
+    .sector_size        = 0x400,
```

Comment:
sector size changed.

Reply:
Do you think it was right before? Where was it used? It looks like the FLM from CMSIS shows one sector size - https://github.com/sarahmarshy/DAPLink/blob/904ea24b4c3ccecd81335bb2be4eb9637028db06/source/target/freescale/kw24d/flash_blob.c#L79-L81